ISPN-12774 Credential store support example for datasource is incomplete (#9084)
infinispan/infinispan: cli/src/main/resources/help/credentials.adoc

--------
`credentials add dbpassword -c changeme -p "secret1234!"` +
Creates a new default credential keystore, if does not already exist, and adds an alias of "dbpassword" for a password of "changeme".

> This command also sets "secret1234!" as the password for the credential keystore, this value should match with the value present in the server configuration like:

Reviewer: ```suggestion
This command also sets "secret1234!" as the password for the credential keystore, which must match the password in the server configuration:
```

 `<clear-text-credential clear-text="secret1234!"/>`

> `credentials ls -p "secret1234!"` +`

Reviewer: ```suggestion
`credentials ls -p "secret1234!"` +
```
